jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -374,16 +374,22 @@ public Connection getConnection(String propertyKey, InterpreterContext interpret
                 if (lastIndexOfUrl == -1) {
                   lastIndexOfUrl = connectionUrl.length();
                 }
-                connectionUrl.insert(lastIndexOfUrl, ";hive.server2.proxy.user=" + user + ";");
+                if (!property.getProperty("hive.proxy.user").equals("false")){
```

Comment:
Just realised, can you do a null check for this, if the property key doesn't exists, this will throw NPE.